inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/job/JobWrapper.java

```diff
@@ -169,4 +170,11 @@ public void addCallbacks() {
     public List<Task> getAllTasks() {
         return allTasks;
     }
+
```

Comment:
Suggest adding Java doc for public methods.

Reply:
done